README.md

```diff
@@ -30,14 +30,18 @@ myCreds =
   { appName = "Your_APP_Name"
   , appId = "your_app_id"
   , appSecret = "xxxxxxxxxxxxxxxxx"
+  , appSecretProofVerification = False
```

Comment:
I don't think the code will work right now as you haven't done these changes in `Credentials` type in `Types.hs`.